learn/guides/visual-studio-code-extension/debugging.md

```diff
@@ -5,7 +5,7 @@ description: Describes debugging functionalities provided by Ballerina in Visual
 keywords: ballerina debugging, ballerina debug, ballerina debugger, ballerina vscode
 permalink: /learn/visual-studio-code-extension/debugging/
 active: debugging
-intro: The VS Code Ballerina extension allows you to either run your Ballerina program (without debugging) or debug them easily by launching its debugger. 
+intro: The VS Code Ballerina language extension comes with in-built debugging capabilities and provides the same debugging experience as the conventional VS Code Debugging.
```

Comment:
```suggestion
intro: The VS Code Ballerina language extension comes with in-built debugging capabilities and provides the same debugging experience as the conventional VS Code Debugging.
```
Shall we say `Visual Stucio Code` here as this is the intro?

Reply:
Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/3385/commits/903828a68d9c603a8a3fec0a6a5ee665c1bcecac